.gitignore

```diff
@@ -76,4 +76,5 @@ buildNumber.properties
 *.iml
 
 ### Mac Files ###
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+*.js
```

Comment:
@DeepaliAgrwl : why .js have been addeed to gitignore ?

Reply:
@rishabh-behl this is committed by mistake. Reverted back.